src/databricks/labs/ucx/source_code/linters/dbfs.py

```diff
@@ -91,7 +91,9 @@ def lint_tree(self, tree: Tree) -> Iterable[Advice]:
 
 class FromDbfsFolder(Linter):
     def __init__(self):
-        self._dbfs_prefixes = ["/dbfs/mnt", "dbfs:/", "/mnt/", "/dbfs/", "/"]
+        # valid dbfs prefixes are ''dbfs:/', /dbfs/mnt', '/mnt/', '/dbfs/', '/'
+        # the latter covers all except the former so just check for those 2
+        self._dbfs_prefixes = ["dbfs:/", "/"]
```

Comment:
`/Volumes` is non-DBFS

Reply:
So we should change the original logic ?